app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -57,6 +56,49 @@
         {% endfor %}
       {% endfor %}
     {% else %}
+      {% import "toolkit/summary-table.html" as summary %}
+        {% call summary.mapping_table(
+        row_bold_borders=True,
+        text_large = True,
+        no_bottom_border=True,
+        caption="If you publish today, you must be aware of the following dates:",
+        caption_visible=True,
+        field_headings=[
+          "Date",
+          "Notice",
+        ],
+        field_headings_visible=False
+      ) %}
+        {% call summary.row(no_border=True) %}
+          {{ summary.field_heading_date("Today", scope='row') }}
+          {{ summary.text("Suppliers can apply and ask questions about your requirements.") }}
+        {% endcall %}
+        {% call summary.row(bold_border=True) %}
+          {{ summary.field_heading_date("Before {}".format(dates.questions_close), rowspan='3', scope='row') }}
+          {{ summary.text_bold("Details of your question and answer session") }}
+        {% endcall %}
+        {% call summary.row(no_border=True) %}
+          {{ summary.text("Question and answer session information.") }}
+        {% endcall %}
+        {% call summary.row(no_border=True) %}
+          {{ summary.text_bold("You must hold your question and answer session before {}.".format(dates.questions_close), border=True) }}
+        {% endcall %}
+        {% call summary.row(bold_border=True) %}
+          {{ summary.field_heading_date(dates.questions_close, scope='row') }}
+          {{ summary.text("Suppliers can no longer ask questions.") }}
+        {% endcall %}
+        {% call summary.row(bold_border=True) %}
+          {{ summary.field_heading_date(dates.answers_close, rowspan='2', scope='row') }}
+          {{ summary.text("You must have published answers to all suppliers’ questions.") }}
+        {% endcall %}
+        {% call summary.row(no_border=True) %}
+          {{ summary.text("If dates.answers_close is a Saturday or Sunday, you need to have published all questions and answers by the Friday before.", border=True) }}
```

Comment:
`dates.answers_close` is just part of the string here - needs to be pulled out into a `.format()`